cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -23,6 +23,26 @@
 ]
 
 
+@pytest.fixture
+def set_default(provider, request):
+    """This fixture is used to return paths for provisioning_entry_point, reconfigure_entry_point
+       and retirement_entry_point. The value of 'provisioning_entry_point' is required while
+       creating new catalog item in 'test_service_infra_tenant_quota_with_default_entry_point' test.
+       But other tests does not require these values since those tests takes default values hence
+       providing default value. So in this file, this fixture - 'set_default'
+       must be used in all tests of quota which are related to services where catalog item needs to
+       be created with specific values for these entries.
+    """
+    with_prov = (
+        "/ManageIQ (Locked)/{}/VM/Provisioning/StateMachines/ProvisionRequestApproval/Default"
+        .format(provider.string_name)
+    )
+    default = (
+        "/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogItemInitialization"
+    )
+    return with_prov if request.param else default
```

Comment:
Why not do something like this:
```
if request.param:
    return_value = 'With Provider"
else:
    return_value = "Default"
return return_value
```

Reply:
Thanks @valaparthvi 
But It will not be more readable as you will get to know that, you are only returning ```return_value```.
Also it's not that difficult to read if we followed existing approach. So no readability issue. I would like to go with existing one.

Reply:
You can always add a comment to indicate what type of value is being returned.